src/core/filters/filterregistry.js

```diff
@@ -56,56 +57,48 @@ export default class FilterRegistry {
   }
 
   /**
-   * Gets the static filters as a {@link CombinedFilter} to send to the answers-core
+   * Gets the static filters as a {@link SimpleFilter|CombinedFilter} to send to the answers-core
+   * @returns {CombinedFilter|SimpleFilter|undefined} Returns undefined if no filters with
+   *                                                  filtering logic are present.
    */
   getStaticFilterPayload () {
     const filterNodes = this.getStaticFilterNodes();
-
-    const transformedFilterNodes = filterNodes.map(filterNode => {
-      return this._transformFilter(filterNode);
-    }).filter(filter => filter);
-
-    if (transformedFilterNodes.length > 1) {
-      return {
-        combinator: '$and',
-        filters: transformedFilterNodes
-      };
-    }
-    return transformedFilterNodes[0];
+    return this._transformFilterNodes(filterNodes, FilterCombinator.AND);
   }
 
-  _transformFilter (filterNode) {
-    if (filterNode.children) {
-      return this._recurseCombinedFilterNodes(filterNode.children, filterNode.combinator);
-    }
+  /**
+   * Transforms a filter node {@link CombinedFilterNode} or {@link SimpleFilterNode} to
```

Comment:
Maybe "Transforms a list of filter nodes ..." since this operates on an `Array` of filter node.